src/block/timeline/edit.js

```diff
@@ -92,6 +91,7 @@ const Edit = props => {
 		className,
 		clientId,
 		isSelected,
+		setAttributes,
```

Comment:
Just a tip, no need to use `useBlockSetAttributesContext` here because `setAttributes` is readily available in the edit props :)